src/App.vue

```diff
@@ -924,13 +946,19 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // NB: will throw if API error
     let draftFiling = await LegalServices.fetchFirstOrOnlyFiling(tempId)
 
+    // set filing type and status
     this.setFilingType(draftFiling.header.name)
     this.setFilingStatus(draftFiling.header.status)
+    // *** TODO: remove after testing
+    if (draftFiling.header.status === FilingStatus.CHANGE_REQUESTED) {
+      this.setFilingStatus(FilingStatus.APPROVED)
+    }
```

Comment:
This is a temporary override for testing. This PR breaks this filing in other ways so I think this is OK to merge into main branch (Dev, etc).

("Change Requested" is only used for continuation in applications, which isn't released to Prod yet, so this change should not affect anything else.)